Dangerfile

```diff
@@ -48,7 +48,7 @@ module VSPDanger
   class ChangeLimiter
     EXCLUSIONS = %w[
       *.csv *.json *.tsv *.txt Gemfile.lock app/swagger modules/mobile/docs spec/fixtures/ spec/support/vcr_cassettes/
-      modules/mobile/spec/support/vcr_cassettes/ db/seeds
+      modules/mobile/spec/support/vcr_cassettes/ db/seeds modules/vaos/app/docs
```

Comment:
If the size of the PR is just a warning this change is probably best to leave out.

Reply:
It was a recommendation made a member of the vfs platform team.

Reply:
good, just wanted to make sure the platform team would complain about it.